torchao/experimental/packed_linear_int8_dynamic_activation_intx_weight_layout.py

```diff
@@ -312,113 +314,35 @@ def _impl_2d_aten(input_tensor, weight_tensor):
 
     target = weight_tensor.tensor_impl.get_layout().target
 
+    if weight_tensor.tensor_impl.get_layout().has_bias:
+        assert (
+            bias is None
+        ), "bias should be None because it is already packed with the weights (has_bias=True)"
```

Comment:
nit: `if: assert`; also fine with leaving it as-is for legibility

```suggestion
      assert (
          not weight_tensor.tensor_impl.get_layout().has_bias or bias is None
      ), "bias should be None because it is already packed with the weights (has_bias=True)"
```